app/views/sessions/prompt.html.erb

```diff
@@ -0,0 +1,40 @@
+<% @title = t('multifactor_authentication') %>
+
+<% if @user.mfa_enabled? %>
+  <%= form_tag mfa_create_session_path, method: :post do %>
+    <div class="text_field">
+      <%= label_tag :otp, t('multifactor_auths.otp_code'), class: 'form__label' %>
+      <%= text_field_tag :otp, '', class: 'form__input', autofocus: true, autocomplete: :off %>
+    </div>
+    <div class="form_bottom">
+      <%= submit_tag t('sign_in'), data: { disable_with: t('form_disable_with') }, class: 'form__submit' %>
+    </div>
+  <% end %>
+<% end %>
+
+<% if @user.webauthn_credentials.any? %>
```

Comment:
I guess people could still have mfa disabled and some webauthn credentials leftover, or should we delete the webauthn credentials when disabling mfa?